vocab_files/observable_property_concepts/invertebrate-substrate-type.ttl

```diff
@@ -14,7 +14,7 @@ making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     skos:exactMatch <http://linked.data.gov.au/def/tern-cv/0209017c-b3eb-41bc-bf21-38c10c5ee063> ;
     skos:prefLabel "invertebrate substrate type" ;
```

Comment:
How come this is invertebrate substrate type? Substrate type is only concerned with the soil.

Reply:
This observable property is probably the same as https://linked.data.gov.au/def/test/dawe-cv/75f55bf0-6972-42ba-ad46-7e24f91e8f6a.

Reply:
Yes, they are the same, thanks for clarifying that! How about creating another issue for this and fixing it in another PR?

Reply:
Yeah sure.